app/controllers/api/v1/products_controller.rb

```diff
@@ -0,0 +1,11 @@
+class Api::V1::ProductsController < ApplicationController
+  def index
+    @products = Product.all
+
+    if @products
+      render json: {status: "SUCCESS", message: "Fetched all the products successfully", data: @products}, status: :ok
+    else
+      render json: @products.errors, status: :bad_request
```

Comment:
What if there are no products, it will still show error, it should return empty array in that case.